### PR TITLE
api adjustments based on virtual paths

### PR DESCRIPTION
**Pull request by WarrenMfg**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ const router = require('./router.js');
 app.use(morgan('dev'));
 app.use(bodyParser.json());
 app.use('/api/about', router);
+app.use('/:id', express.static(__dirname + '/../public'));
```
Comment (incogitance) [inline server/index.js:10]: I am so glad we cleared this up early on...

File: server/index.js [hunk 2]
```diff
@@ -7,7 +7,6 @@ const router = require('./router.js');
 app.use(morgan('dev'));
 app.use(bodyParser.json());
 app.use('/api/about', router);
```
Comment (incogitance) [inline server/index.js:9]: Should we make sure everybody is on the same page of /api/<[module]>  in the next standup meeting?

**Review by incogitance — APPROVED** [review]
looks fine... Great that you are ahead in the front-end department.

File: server/index.js (hunk 2 above)
Comment (WarrenMfg) [reply server/index.js:9]: Yeah for sure

**Pull request merged**